app/views/common/os_selection/_pxe_loader.html.erb

```diff
@@ -0,0 +1,10 @@
+<%= fields_for item do |f| %>
+  <% selected_attr = f.object.new_record? && item.hostgroup.pxe_loader.nil? ? { :selected => @operatingsystem.preferred_loader } : {} -%>
+  <%= selectable_f f, :pxe_loader,
+    @operatingsystem.available_loaders.to_a, {
```

Comment:
As I user I'd prefer to just select "Grub UEFI" and get the 32/64bit info from host's architecture field.